build_web_compilers/lib/src/platforms.dart

```diff
@@ -6,6 +6,7 @@ import 'package:build_modules/build_modules.dart';
 
 const _libraries = [
```

Comment:
My brain always goes to "this should be a `Set`" but no biggy...

Reply:
I mean, you aren't wrong, but it also doesn't matter here in practice :)

Reply:
Oh, I know. 😉